cinch/roles/certificate_authority/tasks/main.yml

```diff
@@ -1,7 +1,20 @@
-- name: install certificate authority
+- name: install certificate authority from URL
   get_url:
     url: "{{ item }}"
     dest: /etc/pki/ca-trust/source/anchors/
+    owner: root
+    group: root
+    mode: 0644
     validate_certs: "{{ certificate_authority_validate_certs }}"
   with_items: "{{ certificate_authority_urls }}"
   notify: install certificate authority
```

Comment:
We've decided to go ahead and add **become: true** for the places that need it in spite of what is contained in site.yml or other places where this is not explicitly set.  This will be the standard from now on.